clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -103,6 +103,14 @@
         Utils.join(SecurityProtocol.names(), ", ") + ".";
     public static final String DEFAULT_SECURITY_PROTOCOL = "PLAINTEXT";
 
+    public static final String SOCKET_CONNECTION_SETUP_TIMEOUT_MS_CONFIG = "socket.connection.setup.timeout.ms";
+    public static final String SOCKET_CONNECTION_SETUP_TIMEOUT_MS_DOC = "The amount of time the client will wait for the socket connection to be established. If the connection is not built before the timeout elapses, clients will close the socket channel.";
+    public static final Long SOCKET_CONNECTION_SETUP_TIMEOUT_MS_DEFAULT = 10 * 1000L;
```

Comment:
We should be consistent with the naming above for `DEFAULT_SECURITY_PROTOCOL` and name this `DEFAULT_SOCKET_CONNECTION_SETUP_TIMEOUT_MS`. We seem to use xxx_DEFAULT in some places, but common configs here and in SslConfigs and SaslConfigs use DEFAULT_xxx, so let's stick to that.

Reply:
Sure. Refactored.